src/lib.rs

```diff
@@ -142,6 +232,26 @@ pub struct PanCam {
     /// If present, the orthographic projection's scale will be clamped at
     /// this value when zooming out.
     pub max_scale: Option<f32>,
+    /// The minimum global x of the camera window. Uses global pixel positions
```

Comment:
Don't these use local positions not global positions (Transform vs GlobalTransform)? What happens if the camera is parented under another transform? For what it's worth, I'd rather they used local coordinates than global, so if I'm not mistaken, I'd rather keep the behavior and change the documentation/naming (`min_x`).

Also, I feel calling the units pixels is probably also misleading, that's only true for specific scales and many applications don't have a concept of pixels either. Maybe just keep it agnostic and call it `minimum x position of the camera window`?